sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -146,7 +146,3 @@ select from_json('{"time":"26/October/2015"}', 'time Timestamp', map('timestampF
 select from_json('{"date":"26/October/2015"}', 'date Date', map('dateFormat', 'dd/MMMMM/yyyy'));
 select from_csv('26/October/2015', 'time Timestamp', map('timestampFormat', 'dd/MMMMM/yyyy'));
 select from_csv('26/October/2015', 'date Date', map('dateFormat', 'dd/MMMMM/yyyy'));
-
-select from_unixtime(1, 'yyyyyyyyyyy-MM-dd');
```

Comment:
I checked these tests which are added to test cases that exceed 10-'y'. It's safe to remove because now it will fail starting from 7-'y' and  we have already covered these in both datetime-parsing.sql and datetime-formatting.sql